src/guide/code-owners.md

```diff
@@ -4,184 +4,221 @@ This list is intended to make it easier to identify which email list to include
+      * `*/security` – Security
+      * `*/text` – I18n
+      * `*/time` – Core Libs
+      * `*/util` – I18n, Core Libs
+      * `jdk/internal`
+        * `access` – Core Libs, Security
+        * `event` – JFR
+        * `icu` – Core Libs
+        * `javac` – Java Language
+        * `jimage` – Client Libs
+        * `jrtfs` – Core Libs
+        * `loader` – Core Libs
+        * `logger` – Core Libs
+        * `org` – Core Libs
+        * `perf` – Runtime
+        * `platform` – HotSpot
+        * `vm` – HotSpot
+      * `sun/launcher` – Generic List
+      * `META-INF` – Core Libs
+    * `*/conf`
+      * `sdp` – Net
+      * `security` – Security
+    * `*/legal` –
+    * `*/lib/security` – Security
+    * `*/native`
+      * `common` –
+      * `include` – Runtime, Core Libs
+      * `jspawnhelper` – Generic List
+      * `launcher` – Generic List
+      * `libjava` – Core Libs
+      * `libjimage` – Core Libs
+      * `libjli` – Generic List
+      * `libjsig` – HotSpot
+      * `libnet` – Net
+      * `libnio` – NIO
+      * `libosxsecurity` – Security
+      * `libsyslookup` – Core Libs
+      * `libverify` – Runtime
+      * `libzip` – Core Libs
+    * `share/data`
+      * `blockedcertsconverter` – Security
+      * `cacerts` – Security
+      * `currency` – I18n
+      * `lsrdata` – I18n
+      * `publicsuffixlist` – Client Libs
+      * `tzdata` – I18n
+      * `unicodedata` – I18n
+    * `share/man`
+      * `java.1` – Generic List
+      * `keytool.1` – Security
+  * `java.compiler` – Javac
+  * `java.datatransfer` – Client Libs
+  * `java.desktop` – Client Libs
+  * `java.instrument` – Serviceability
+  * `java.logging` – Core Libs
+  * `java.management` – Serviceability
+  * `java.management.rmi` – Serviceability
+  * `java.naming` – Core Libs
+  * `java.net.http` – Net
+  * `java.prefs` – Core Libs
+  * `java.rmi` – Core Libs
+  * `java.scripting` – Generic List
+  * `java.se` – Core Libs
+  * `java.security.jgss` – Security
+  * `java.security.sasl` – Security
+  * `java.smartcardio` – Security
+  * `java.sql` – Core Libs
+  * `java.sql.rowset` – Core Libs
+  * `java.transaction.xa` – Core Libs
+  * `java.xml` – Core Libs
+  * `java.xml.crypto` – Security
+  * `jdk.accessibility` – Client Libs
+  * `jdk.attach` – Serviceability
+  * `jdk.charsets` – I18n, Core Libs
+  * `jdk.compiler` – Javac
+  * `jdk.crypto.cryptoki` – Security
+  * `jdk.crypto.ec` – Security
+  * `jdk.crypto.mscapi` – Security
+  * `jdk.dynalink` – Nashorn
+  * `jdk.editpad` – JShell
+  * `jdk.hotspot.agent` – Serviceability
+  * `jdk.httpserver` – Net
+  * `jdk.incubator.concurrent` – Core Libs
+  * `jdk.incubator.vector` – Compiler
+  * `jdk.internal.ed` – JShell
+  * `jdk.internal.jvmstat` – Serviceability
+  * `jdk.internal.le` – JShell
+  * `jdk.internal.opt` – Generic List
+  * `jdk.internal.vm.ci` – Compiler
+  * `jdk.internal.vm.compiler` – Compiler
+  * `jdk.internal.vm.compiler.management` – Compiler
+  * `jdk.jartool` – Generic List, Security
```

Comment:
jartool (and APIs defined in `java.util.jar`) are both discussed/reviewed in Core Libs